data/frame_meta.go

```diff
@@ -17,7 +17,7 @@ type FrameMeta struct {
 
 	// TypeVersion is the version of the Type property. Versions greater than 0.0 correspond to the dataplane
 	// contract documentation https://github.com/grafana/grafana-plugin-sdk-go/tree/main/data/contract_docs.
-	TypeVersion *FrameTypeVersion `json:"typeVersion,omitempty"`
+	TypeVersion FrameTypeVersion `json:"typeVersion,omitempty"`
```

Comment:
@kylebrandt this makes sense in general to me, but i'm wondering if with this change the versions of things like [heatmaps](https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/contract_docs/heatmap.md#heatmap-buckets-heatmapbuckets) be bumped to `0.1` instead of the default of `0.0`?

Reply:
0.0 means alpha, not in the contract yet. It may have a place in the spec, but is draft and offers no guarantees.

For types that are currently in the spec marked 0.0 , it was because I don't feel I have no hands on experience with either consumers or producers (although I understand the formats). So feel someone else should put a version on it when they are ready.

Reply:
ah ok, makes sense 👍